hyperspy/tests/mva/test_bss.py

```diff
@@ -457,4 +457,4 @@ def test_bss_supported_return_true(self):
 
     @pytest.mark.skipif(not sklearn_installed, reason="sklearn not installed")
     def test_bss_supported_return_false(self):
-        assert self.s.blind_source_separation(return_info=False) is None
+        assert self.s.blind_source_separation(return_info=False) is None
```

Comment:
why has this crept in?

Reply:
Not sure - I think it's windows end of line difference.